nullaway/src/main/java/com/uber/nullaway/handlers/AssertionHandler.java

```diff
@@ -45,17 +45,13 @@ public class AssertionHandler extends BaseNoOpHandler {
   @Override
   public NullnessHint onDataflowVisitMethodInvocation(
       MethodInvocationNode node,
+      Symbol.MethodSymbol callee,
```

Comment:
note that i've adjusted the parameter name in the handlers to keep the diff small

if you want it to be same as in the base interface let me know (and how to call it i.e. callee, symbol, methodSymbol etc.)